index.html

```diff
@@ -1227,8 +1224,7 @@ <h3>Supporting algorithms</h3>
           Let |record:PressureRecord| be |observer|.{{PressureObserver/[[LastRecordMap]]}}[|source|].
         </li>
         <li>
-          If |record|.{{PressureRecord/[[State]]}} is not equal to |state| and [=change in contributing factors is substantial=]
```

Comment:
Is this change related to the rest? It doesn't seem to reference `[=current pressure state=]`.

Reply:
It is. This is done on the global value now instead of per observer.

Reply:
I see. It'd be good to explain this in the commit/PR message, as it's not immediately clear just by looking at the patch.